### PR TITLE
#54 - Checking that layers exist when manifest is added

### PR DESCRIPTION
**Pull request by olegmoz**
Part of #54 
Checking layer blobs exist when manifest is added

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#115](https://github.com/artipie/docker-adapter/issues/115) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/821a5fba-cb18-41fb-aa12-86f51269bf9c); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/test/java/com/artipie/docker/http/ManifestEntityPutTest.java [hunk 1]
```diff
@@ -105,4 +105,9 @@ void shouldPushManifestByDigest() {
             )
         );
     }
+
+    private Flowable<ByteBuffer> manifest() {
+        final byte[] data = "{\"layers\":[]}".getBytes();
```
Comment (carlosmiranda) [inline src/test/java/com/artipie/docker/http/ManifestEntityPutTest.java:110]: @olegmoz let's inline this variable

Comment (olegmoz) [reply src/test/java/com/artipie/docker/http/ManifestEntityPutTest.java:110]: @carlosmiranda done, thanks

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by sereshqua** [comment]
@carlosmiranda please make sure you will find at least 3 issues during next CR, thanks!

**Comment by carlosmiranda** [comment]
@sereshqua yes, thank you

**Comment by sereshqua** [comment]
@0crat quality acceptable

**Pull request merged**